tests/runtest.cmd

```diff
@@ -22,15 +26,18 @@ set __MSBuildBuildArch=x64
 :: is already configured to use that toolset. Otherwise, we will fallback to using the VS2015
 :: toolset if it is installed. Finally, we will fail the script if no supported VS instance
 :: can be found.
-if defined VisualStudioVersion goto :Run
+if defined VisualStudioVersion ( 
+    if not defined __VSVersion echo %__MsgPrefix%Detected Visual Studio %VisualStudioVersion% developer command ^prompt environment
```

Comment:
What is the use of this message? We should still default __VSVersion from the VisualStudioVersion similar to https://github.com/dotnet/corefx/blob/master/src/Native/build-native.cmd#L69.

Reply:
I wanted to have a message which tests for consistency of VS developer command prompt detection and prints result. One of the reasons for some builds failing on my machine was a problem with VS environment detection and unnecessary execution of VsDevCmd.bat for preview VS. Now it is fixed and reports version detected.

In all scripts there is a new block for detecting VS installation and it's version introduced in #14228. I have not changed it - if it should be changed I can do it just let me know.

`if not defined __VSVersion echo` usage is due to the fact that currently multiple scripts contain this detection code and this conditional eliminates duplicate messages from any script called from a top level one.

Reply:
I'm good with the message for now, lets handle cleaning this up and making it consistent as part of the other issues.